benchmarks/appsec_iast_aspects/config.yaml

```diff
@@ -0,0 +1,5 @@
+no_iast: &base_variant
+iast_enabled: 0
+
+iast_enabled: &iast_enabled
+iast_enabled: 1
```

Comment:
How does this work with the scenario.py file?

Reply:
It fills the `iast_enabled` member from the `IAST_Aspect` class (which inherits from the benchmark one). The PR is not yet finished because while it works it doesn't give a report with the granularity I would like and need to investigate how to do it (I moved it back to draft).